### DISCUSSION TITLE
list as a card view

### DISCUSSION THREAD

**Discussion by Jamlee** [community/community#9055] [Other Feature Feedback, Questions, & Ideas]
list as a card view

**Comment by brianlovin** [comment]
Could you share more about what you're looking for here? What would be on the card, and where would those cards be useful to appear on GitHub?

**Reply by Jamlee** [reply]
just **list name**, **list description**, **count of repos** on card.  the list may be not  make the best of space. 😊

![image](https://user-images.githubusercontent.com/5196641/148150493-2776dbf5-3a2d-4cbd-b17d-51915441cac8.png)

**Reply by brianlovin** [reply]
Gotcha. In some cases, Lists can have a description which will render in that empty space, like this:

<img width="934" alt="Screen Shot 2022-01-05 at 09 26 57@2x" src="https://user-images.githubusercontent.com/1923260/148261618-4368a127-d3c3-43e8-9c80-24f57edb2822.png">

But in the case where there are no descriptions, it definitely feels empty! I'll pass this along to the team 😊
